models/stg_google_play__subscriptions_country.sql

```diff
@@ -0,0 +1,39 @@
+{{ config(enabled=var('google_play__using_subscriptions', False)) }}
+
+with base as (
+
+    select * 
+    from {{ ref('stg_google_play__subscriptions_country_tmp') }}
+
+),
+
+fields as (
+
+    select
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(ref('stg_google_play__subscriptions_country_tmp')),
+                staging_columns=get_financial_stats_subscriptions_country_columns()
+            )
+        }}
+        
+    from base
+),
+
+final as (
+    
+    select 
+
+        date as date_day,
+        country,
+        product_id,
+        package_name,
+        -- batch nulls together
```

Comment:
same batch question as above (comparing this comment with the group by comments)